pkg/kubelet/cm/cpumanager/policy_static_test.go

```diff
@@ -622,7 +623,7 @@ func runStaticPolicyTestCase(t *testing.T, testCase staticPolicyTest) {
 	}
 
 	container := &testCase.pod.Spec.Containers[0]
-	err := policy.Allocate(st, testCase.pod, container)
+	err := unwrapErr(policy.Allocate(st, testCase.pod, container))
```

Comment:
What is the reason to return an `error` and then immediately unwrap it?

Reply:
because the tests check errors for equality (e.g. they *not* use `errors.As` or `errors.Is`) and this is the quickest fix still being (IMO) (barely?) good enough.
We should probably clean the tests to use `errors`, but this is a larger endeavor. If reviewers feel otherwise though, fair enough, I can prepend this cleanup to this PR.

Reply:
Thank you for the explanation.
Yes we probably should wait from the other reviewers comments on that.

Reply:
I would prefer do the unwrap at the point of comparison, not at the return site.